upgrades/upgrade_testing_framework/steps/step_expectation_selector.py

```diff
@@ -0,0 +1,16 @@
+from upgrade_testing_framework.core.framework_step import FrameworkStep
+
+class ExpectationSelector(FrameworkStep):
```

Comment:
In general, I'd prefer we name our FrameworkSteps as an action-to-be-performed rather than a thing-doing-work.  So, how about `SelectExpectations` instead?  The idea is the high-level log output should be as descriptive as possible and the step name is a key part of understanding that.

Reply:
Agreed will add